README.md

```diff
@@ -1 +1,72 @@
-# StoopidProject
\ No newline at end of file
+# JIRA
+Our JIRA can be found here: https://georgelosttoabot.atlassian.net/browse/SP-3
+We use JIRA to keep track of which task each person is working on. This way, we can easily see who is working on which task at any time. You can log a task JIRA on the epic to say which task you are working on. Within a task JIRA, you can log subtasks. No coding task should take longer than 8 hours of coding. If you predict a task will take longer than 8 hours of coding, then you should break down the task into multiple subtasks. Your GitHub branch is named after your subtask coding JIRa. If you don't need a subtask JIRA, then you can name your GitHub branch after the task jira.
+
+# Software Development Process
+1) Brainstorm a task to work on.
+2) Log JIRAS for each task that you came up with and assign the JIRAS to yourself.
+3) Place the JIRA you are about to start working on as 'In Progress.'
+4) Create your GitHub branch by branching off of master. Name this branch after your JIRA name.
+5) Code on your branch until you feel that your coding is complete. Make sure you have tested it and everything works.
+6) Pull down the latest code from master into your local master branch. Merge the code from your local master into your development branch if your branch doesn't already have the latest master code. Resolve merge conflicts if there are any.
+7) Go to GitHub and post a pull request from your branch to master.
+   -link your JIRA on the pull request
+   -make sure your JIRA is documented to include what you did in your task  
+8) Teammates should look at the files changed sectino of the pull request and make comments on how to improve the code and fix potential bugs. This is your chance to review the code of your teammates and provide feedback. After this, the teammates should check out the pull request branch and pull down the code and test the application. They should run a mvn clean install and try using the application to make sure it works.
+9) After all teammates approve of the pull request, you can press the green button to merge the code from your development branch to master.
```

Comment:
instead of "after all teammates approve" change to "After 2 or more teammates approve"

Reply:
fixed